scripts/snyk-test.js

```diff
@@ -17,7 +17,7 @@ async function snykTest(cwd) {
       await execFile(
         'npx',
         [
-          'snyk',
+          'snyk@latest',
```

Comment:
Just curious: should it be a specific dependency of snyk we pin here (or install in dependencies of scripts even) to avoid more unexpected issues, or is it required that we always run against latest possible?